types/tslint.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
TSLint is deprecated, see: https://github.com/palantir/tslint/issues/4534

ESLint with `@typescript-eslint` should be used instead

Reply:
You know come to think of it I probably don't need that file to be part of this PR anyway. I was mainly using it for my own convenience. I'll see about omitting it. Anyway, great point about the deprecation.

Reply:
Oh, I actually just tried removing it.  Looks like `dtslint` complains more without it.

This is really more of a `dtslint` issue as described here: https://github.com/microsoft/dtslint/issues/214